airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1326,3 +1326,90 @@ definitions:
       $parameters:
         type: object
         additionalProperties: true
+interpolation:
+  variables:
```

Comment:
@flash1293 FYI I added the list of variables that can be part of the context since they're not documented anywhere

Reply:
Nice, good catch!